docs/aspnetcore-mvc.md

```diff
@@ -86,6 +86,57 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
+
+## Configuration for cloud services or load balancers
+
+If you plan to deploy your application to a cloud service, you may need to do additional customization. Cloud services usually create similar environments in which applications run inside a Docker container without using HTTPS. The front host or load balancer that the user has direct access to may or may not use HTTPS by-default. 
+
+Cloud services are vary in configuration, but general rules are:
+
+* Make sure Load Balancer or front host uses HTTPS. 
+
+This may be the default or may require some amount of fiddling as in case of AWS.
+
+
+* Your Application should be aware of HTTPS scheme used by user despite the App itself is using HTTP. 
```

Comment:
I'm not clear what this sentence is trying to communicate.  **Application** should not be capitalized and **App** should be spelled out and not capitalized.